packages/salesforcedx-vscode-core/test/vscode-integration/orgBrowser/orgMetadata.test.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as fs from 'fs';
+import * as path from 'path';
+import { SinonStub, stub } from 'sinon';
+import { isNullOrUndefined } from 'util';
+import { nls } from '../../../src/messages';
+import {
+  buildTypesList,
+  ForceDescribeMetadataExecutor,
+  getMetadataTypesPath
+} from '../../../src/orgBrowser';
+import {
+  getRootWorkspacePath,
+  hasRootWorkspace,
+  OrgAuthInfo
+} from '../../../src/util';
+
+describe('Force Describe Metadata', () => {
+  it('Should build describe metadata command', async () => {
+    const outputPath = 'outputPath';
+    const forceDescribeMetadataExec = new ForceDescribeMetadataExecutor(
+      outputPath
+    );
+    const forceDescribeMetadataCmd = forceDescribeMetadataExec.build({});
+    expect(forceDescribeMetadataCmd.toCommand()).to.equal(
+      `sfdx force:mdapi:describemetadata --json --loglevel fatal -f ${outputPath}`
+    );
+  });
+});
+
+// tslint:disable:no-unused-expression
+describe('getMetadataTypesPath', () => {
+  let getDefaultUsernameStub: SinonStub;
+  let getUsernameStub: SinonStub;
+  const rootWorkspacePath = getRootWorkspacePath();
+  beforeEach(() => {
+    getDefaultUsernameStub = stub(OrgAuthInfo, 'getDefaultUsernameOrAlias');
+    getUsernameStub = stub(OrgAuthInfo, 'getUsername');
+  });
+  afterEach(() => {
+    getDefaultUsernameStub.restore();
+    getUsernameStub.restore();
+  });
+
+  it('should return the path for a given username', async () => {
+    getDefaultUsernameStub.returns('defaultUsername');
```

Comment:
This should always return an username in email format e.g. `test-audskbqx3kttjb@example.com`

Reply:
I kept the `getDefaultUsernameStub` as `defaultAlias` because it can return an alias or a username, but I updated the `getUsername` stub to be in an email format, since that ensures we only have the username.